### ISSUE TITLE
Mentions example page is not working.

### ISSUE THREAD

**Issue by priteshjha4u** [ianstormtaylor/slate#2770]
#### Do you want to request a _feature_ or report a _bug_?
Bug

#### What's the current behavior?
Mentions example is not working. App crashes when navigating to mentions page. I guess it is appearing after #2747 is merged, since the error message is about set annotations.

```
TypeError: n.editorRef.current.setannotations is not a function
```
https://www.slatejs.org/#/mentions

![error](https://user-images.githubusercontent.com/2283282/57629127-e3ee7a80-75b8-11e9-84b3-bdd398401448.png)

#### What's the expected behavior?
It should work as expected.

**Comment by XiaocongDong** [comment]
Have the same issue here, any information on this?

**Comment by adjourn** [comment]
`setAnnotations` command (or `setannotations` here) doesn't exist. Not sure if placeholder or.. I could try to hack something together and submit a PR but I'd rather wait what @ianstormtaylor has to say about this.

**Comment by nadeemshaik** [comment]
@priteshjha4u @XiaocongDong This should be fixed after #2877
